Sources/Benchmark/BenchmarkMetric.swift

```diff
@@ -31,12 +31,12 @@ public enum BenchmarkMetric: Hashable, Equatable, Codable, CustomStringConvertib
     case peakMemoryResidentDelta
     /// Measure virtual memory usage - sampled during runtime
     case peakMemoryVirtual
-    /// Number of small malloc calls
-    case mallocCountSmall
-    /// Number of large malloc calls
-    case mallocCountLarge
-    /// Number of small+large mallocs
+    /// Number of total mallocs
     case mallocCountTotal
+    /// Number of totatl free calls
+    case freeCountTotal
```

Comment:
I think we should keep the small and large malloc count metrics, I would suggest that we put the threshold for what is large/small to match the page size (and document it as such, so e.g. on macOS/ARM it'd be 16KB, while on Linux x86 it'd be 4K). Removing these metrics would be unnecessarily source breaking.